### DISCUSSION TITLE
Running the launcher replication-orchestrator failed

### DISCUSSION THREAD

**Discussion by LucasChies** [airbytehq/airbyte#36824] [Connector Questions]
**Environment**
Airbyte version: 0.53.1
OS Version / Instance: Standard_D8ads_v5 on Azure AKS
Deployment: Kubernetes (Container Orchestrator)
Step where error happened: Sync job

**Error Message**
message='io.temporal.serviceclient.CheckedExceptionWrapper: io.airbyte.workers.exception.WorkerException: Running the launcher replication-orchestrator failed', type='java.lang.RuntimeException', nonRetryable=false

This error happens whatever connection I setup. Someone alrady know this issue?
I already attach the log file.

[default_workspace_job_6_attempt_1_txt.txt](https://github.com/airbytehq/airbyte/files/14866699/default_workspace_job_6_attempt_1_txt.txt)

**Comment by sw-lkinsella** [comment]
Did you ever manage to figure this out? We're seeing a similar issue this morning.

[a comment by an automated account is not shown]
